go/pvl/helpers.go

```diff
@@ -27,7 +27,7 @@ func pvlSubstitute(template string, state PvlScriptState, match []string) (strin
 
 	var outerr libkb.ProofError
 	// Regex to find %{name} occurrences.
-	re := regexp.MustCompile("%\\{[\\w]+\\}")
+	re := regexp.MustCompile("%\\{[\\w]*\\}")
```

Comment:
Let's add a quick comment about why we match `*` here (for better error reporting).